flutter_local_notifications/example/lib/main.dart

```diff
@@ -2347,6 +2371,57 @@ class _HomePageState extends State<HomePage> {
             ));
   }
 
+  Future<void> _createNotificationChannelWithDndBypass() async {
+    const AndroidNotificationChannel androidNotificationChannel =
+        AndroidNotificationChannel('your channel id 3', 'your channel name 3',
+            description: 'your channel description 3',
+            bypassDnd: true,
+            importance: Importance.max);
+
+    final AndroidFlutterLocalNotificationsPlugin? androidPlugin =
+        flutterLocalNotificationsPlugin.resolvePlatformSpecificImplementation<
+            AndroidFlutterLocalNotificationsPlugin>();
+
+    final bool? hasPolicyAccess =
+        await androidPlugin?.hasNotificationPolicyAccess();
+    if (hasPolicyAccess == true) {
```

Comment:
```suggestion
    if (hasPolicyAccess ?? false) {
```

Spotted a logic issue as I was testing more thoroughly. This is meant to direct users to settings if they hadn't granted access, which means `hasNotificationPolicyAccess()` would actually return `false`. Unless this is fixed, those running the example for the first time will actually never be able to create a channel that bypass DnD